### PR TITLE
General washing fixes

### PR DESCRIPTION
**Pull request by WarlockD**
## About The Pull Request
A few bugs with washed.  The sink didn't really wash properly and the suit would be skipped when you washed in the shower.  Also cleaned up the proc a bit on a few edge cases.

## Why It's Good For The Game
Who doesn't want to be clean?

## Changelog
:cl:
fix: Your suit now washes under the shower
fix: Sink now cleans properly
/:cl:

### CODE REVIEW

File: code/game/atoms.dm [hunk 1]
```diff
@@ -945,19 +945,19 @@
...
 
 	var/datum/component/radioactive/healthy_green_glow = GetComponent(/datum/component/radioactive)
-	if(!healthy_green_glow || QDELETED(healthy_green_glow))
-		return
-	var/strength = healthy_green_glow.strength
-	if(strength <= RAD_BACKGROUND_RADIATION)
-		qdel(healthy_green_glow)
-		return
-	healthy_green_glow.strength -= max(0, (healthy_green_glow.strength - (RAD_BACKGROUND_RADIATION * 2)) * 0.2)
+	if(healthy_green_glow && !QDELETED(healthy_green_glow))
```
Comment (lordpidey) [inline code/game/atoms.dm:953]: No need to check if the object exists, `QDELETED` already does that.

**Review by lordpidey — CHANGES_REQUESTED** [review]

File: code/game/atoms.dm (hunk 1 above)
Comment (WarlockD) [reply code/game/atoms.dm:953]: Ah ok, it was in the original code

File: code/game/atoms.dm [hunk 2]
```diff
@@ -945,19 +945,19 @@
...
 
 	var/datum/component/radioactive/healthy_green_glow = GetComponent(/datum/component/radioactive)
-	if(!healthy_green_glow || QDELETED(healthy_green_glow))
-		return
-	var/strength = healthy_green_glow.strength
-	if(strength <= RAD_BACKGROUND_RADIATION)
-		qdel(healthy_green_glow)
-		return
-	healthy_green_glow.strength -= max(0, (healthy_green_glow.strength - (RAD_BACKGROUND_RADIATION * 2)) * 0.2)
+	if(healthy_green_glow)
```
Comment (lordpidey) [inline code/game/atoms.dm:953]: No, you want `if(!QDELETED(healthy_green_glow)`

**Review by lordpidey — CHANGES_REQUESTED** [review]

**Comment by LemonInTheDark** [comment]
good damn work

**Review by lordpidey — APPROVED** [review]

**Pull request merged**